apps/hash-graph/.justfile

```diff
@@ -31,7 +31,40 @@ deployment-up *arguments: docker-build
 deployment-down *arguments:
   @just yarn external-services down {{arguments}}
 
+[private]
+test *arguments:
+  @just deployment-up
+  @just --justfile {{repo}}/.justfile test {{arguments}}
+  cargo test -p graph-benches --benches --profile {{profile}} {{arguments}}
+  @just deployment-up graph --wait
+  @just yarn httpyac send --all {{repo}}/apps/hash-graph/tests/rest-test.http
+  @just deployment-down
+  @just generate-openapi-client
+
+[private]
+coverage *arguments:
+  @just deployment-up
+  cargo llvm-cov --workspace --all-features --all-targets {{arguments}}
```

Comment:
Any reason why this is different from the root justfile? I see that the root rule uses `nextest`, is there a reason why this wasn't working with the graph?

Reply:
`nextest` does not play well with `criterion`

```
error: creating test list failed

Caused by:
  for `graph-benches::bench/read_scaling`, command `/Users/ghost/Development/hash/apps/hash-graph/target/llvm-cov-target/debug/deps/read_scaling-3150a3fb42776272 --list --format terse` exited with code 2
--- stdout:

--- stderr:
Gnuplot not found, using plotters backend
error: Found argument '--format' which wasn't expected, or isn't valid in this context

        If you tried to supply `--format` as a value rather than a flag, use `-- --format`

USAGE:
    read_scaling-3150a3fb42776272 --list

For more information try --help

---
error: process didn't exit successfully: `/Users/ghost/.rustup/toolchains/nightly-2023-04-04-aarch64-apple-darwin/bin/cargo nextest run --manifest-path /Users/ghost/Development/hash/apps/hash-graph/Cargo.toml --target-dir /Users/ghost/Development/hash/apps/hash-graph/target/llvm-cov-target --workspace --all-features --all-targets` (exit status: 104)
cargo llvm-cov nextest     220.75s user 33.28s system 773% cpu 32.861 total
```